src/core.js

```diff
@@ -225,6 +225,10 @@ define([
             analytics.workflow("Clicked on easy reference popover's link to show in tree");
         });
 
+        $(window).on('hashchange', function () {
+            _this.selectSomethingOrHideProperties(true, window.location.hash);
```

Comment:
I wasn't able to write a test for this because the `onhashchange` event seems to trigger some arbitrary amount of time after the hash actually changes. Short of putting some kind of sleep statement in the test, I just made sure that this worked manually in the UI.